### ISSUE TITLE
Problem with export

### ISSUE THREAD

**Issue by athamour1** [silverslade/aframe_blender_exporter#16]
# Problem with export
I am using blender 2.8.3(with Python version 3.7.4), version 8 of the addon, and the Manjaro-Gnome os with 5.26 kernel .(i also have tried newer versions and version 7 of the addon, and every time I have the same result). Also, I don't use the pre-baked lighting and my settings for the export are the defaults except for the export directory.
![image](https://user-images.githubusercontent.com/39333862/155295146-c62bda85-0dca-40f0-b11a-98f7e6d0a130.png)
Also, i don't know if this is relevant but when I was trying to import the gltf model inside the aframe entity i was getting an error about custom UV lighting and that why i tried to use this addon.

**Comment by silverslade** [comment]
Hi!
The latest exporter version is tested on blender 3.0 or above.
Your issues seems to be related to the version of the gltf exporter.
Please check out it, this should be GLTF 2.0 (since version 1.7.33 or above) 

![immagine](https://user-images.githubusercontent.com/8178513/155794177-986357ef-847a-42ca-b2af-6cab82c625ab.png)

**Comment by elolelo** [comment]
<img width="1440" alt="Screenshot 2022-09-18 at 21 04 56" src="https://user-images.githubusercontent.com/32453496/190924405-b6e816d5-2988-4a04-b6e0-9bafd81827b1.png">

Hi, 

I am on mac running the 3.3.0 version of blender and getting a similar error. I am new to blender, I am not sure if there is anything I am doing wrong to get the error. I can't export to aframe. I was following [this tutorial](https://youtu.be/smFPbUk5SCE) but failed to continue after not being able to export to aframe. I'd appreciate suggestions on how to make this error go away.

**Comment by AsciiSmoke** [comment]
I too am new to both Blender and A-Frame. I'm seeing the same error as OP.
I have glTF 2.0 format and the A-Frame exporter enabled and I'm running Blender 3.3 on Windows 10.
I've also installed the latest python release.

I was previously running Blender from Steam (installed via the Steam store). I've tried removing and installing the program directly from Blender's website just in case there was some permission issue with the Steam version.

![Screenshot 2022-09-23 124540](https://user-images.githubusercontent.com/12941199/191953884-db4dc4c0-a9a1-452e-aaee-b24cd3b544f7.png)

Also, if I click 'Start Serving', then click 'Open Preview', I get a directory listing page, 
![image](https://user-images.githubusercontent.com/12941199/191954012-12ca26fa-d7bd-4923-bdd6-9f4749e2ad4b.png)

In case it helps, the View3D viewport window seems to permanently say 'exporting...' even before you click export.
![image](https://user-images.githubusercontent.com/12941199/191959245-b128d776-84ee-41b9-a1d7-aac75722d079.png)
